prober/tls_test.go

```diff
@@ -140,16 +141,60 @@ func (s *CRLServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Write(s.crlBytes)
 }
 
-func TestCRL(t *testing.T) {
-	// Generate CA key and self-signed CA cert
-	caKey, err := rsa.GenerateKey(rand.Reader, 4096)
```

Comment:
RSA is slow. At first I just changed it to ECDSA which was much faster, but still a second on a super fast machine.

Now it's more like zero, baking in some keys.